packages/hierarchy-browser/src/CommunityCard/CommunityCard.tsx

```diff
@@ -0,0 +1,173 @@
+			() => hierachyDataProvider.getNeighborsAtLevel(community.communityId),
+			[hierachyDataProvider, community],
+		)
+
+		useLayoutEffect(() => {
+			if (dataProvider) {
+				dataProvider.neighborCommunities = neigborCommunities
+			}
+		}, [dataProvider, neigborCommunities])
+		const [
+			entities,
+			isLoading,
+			loadMore,
+			hasMore,
+			isOpen,
+			toggleOpen,
+			filterProps,
+		] = useCommunityData(isOpenProp, maxLevel, dataProvider)
+
+		const [
+			adjacentCommunities,
+			isAdjacentEntitiesLoading,
+		] = useAdjacentCommunityData(isOpen, dataProvider)
+
+		const [
+			setEdgeSelection,
+			loadMoreEntities,
+			moreEntitiesToLoad,
+			edgeEntities,
+			selectedCommunityEdge,
+			clearCurrentSelection,
+		] = useEdgeSelection(dataProvider)
+		const sizePercent = useCommunitySizePercent(community, maxSize)
+		const contentStyle = useContainerStyle(isOpen, entities.length > 0)
+
+		const loadingElement = useMemo(
+			() => (isLoading ? <Spinner label={ENTITY_LOADER_MSG} /> : null),
+			[isLoading],
+		)
+
+		const colorStyle = useThemesAccentStyle(isOpen)
+
+		return (
+			<div>
+				<CommunityOverview
+					community={community}
+					onToggleOpen={toggleOpen}
+					incrementLevel={incrementLevel}
+					sizePercent={sizePercent}
+					filterProps={filterProps}
+					getEntityCallback={loadMore}
+					level={level}
+				/>
+				<Flex>
+					<Content style={contentStyle}>
+						{entities && entities.length > 0 ? (
+							<ScrollArea loadMore={loadMore} hasMore={hasMore}>
+								<CommunityTable
+									entities={entities}
+									minimize={false}
+									communityId={community.communityId}
+								/>
+							</ScrollArea>
+						) : null}
+						{loadingElement}
+						<EmptyEntityList
+							filterProps={filterProps}
+							entities={entities}
+							isLoading={isLoading}
+						/>
+					</Content>
+					<Spacer style={colorStyle}></Spacer>
+					{adjacentCommunities && adjacentCommunities.length > 0 ? (
+						<Content style={contentStyle}>
+							<CommunityEdgeList
+								edges={adjacentCommunities}
+								selectedEdge={selectedCommunityEdge}
+								onEdgeClick={setEdgeSelection}
+								clearCurrentSelection={clearCurrentSelection}
+							/>
+						</Content>
+					) : null}
+					{isAdjacentEntitiesLoading ||
+					(edgeEntities && edgeEntities.length > 0) ? (
+						<Content style={contentStyle}>
+							{edgeEntities && edgeEntities.length > 0 ? (
+								<ScrollArea
+									loadMore={loadMoreEntities}
+									hasMore={moreEntitiesToLoad}
+								>
+									<CommunityTable
+										entities={edgeEntities}
+										minimize={false}
+										communityId={selectedCommunityEdge?.communityId}
+									/>
+								</ScrollArea>
+							) : null}
+							{isAdjacentEntitiesLoading ? (
+								<Spinner label={ENTITY_LOADER_MSG} />
+							) : null}
+						</Content>
+					) : null}
+				</Flex>
+			</div>
+		)
+	},
+)
+CommunityCard.displayName = 'CommunityCard'
```

Comment:
Does this actually work? I ran into issues where this didn't result in display names in the React Components panel of the DevTools, but maybe they fixed that